Sat Jun  3 23:43:17 CEST 2006  VMiklos <vmiklos@frugalware.org>
  * xconfig: typo fix
  when the user clicked we assumed he didn't and vica versa.. ;-)
diff -rN -u old-frugalwareutils/xconfig/xconfig.c new-frugalwareutils/xconfig/xconfig.c
--- old-frugalwareutils/xconfig/xconfig.c	2006-06-04 00:00:13.000000000 +0200
+++ new-frugalwareutils/xconfig/xconfig.c	2006-06-04 00:00:13.000000000 +0200
@@ -83,7 +83,7 @@
 		ret = fwx_dotest();
 		init_dialog(input, dialog_state.output);
 		dialog_backtitle(_("X configuration"));
-		if(ret)
+		if(!ret)
 			break;
 	}
 	unlink("/root/xorg.conf.new");

